src/Symfony/Component/Serializer/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ CHANGELOG
 * added support for XML comment encoding (encoding `['#comment' => ' foo ']` results `<!-- foo -->`)
 * added optional `int[] $encoderIgnoredNodeTypes` argument to `XmlEncoder::__construct` to configure node types to be
   ignored during encoding.
+* the `AbstractNormalizer::handleCircularReference()` method will have two more arguments : `$format` and 
+  `$context` in version 5.0, not defining it is deprecated since version 4.2.
```

Comment:
not defining them

Reply:
thanks @nicolas-grekas fixed the typo and revert unit tests :)